testutil/datagen/btcstaking.go

```diff
@@ -121,7 +121,7 @@ func GenRandomBTCDelegation(
 	if err != nil {
 		return nil, err
 	}
-	staker := GenRandomAccount()
+	staker := GenRandomAccountWithPrefix("bbn") // Staker address is always Babylon's
```

Comment:
We can use [Bech32PrefixAccAddr](https://github.com/babylonlabs-io/babylon/blob/c26151a04894976132cf8d343225298c5248083c/app/params/config.go#L20-L21) so that we don't need to hardcode

Reply:
We can, as long as this is not chain specific. Will try it.